Frameworks/UIKit/UIView.mm

```diff
@@ -3539,37 +3543,30 @@ - (UIView*)snapshotViewAfterScreenUpdates:(BOOL)afterUpdates {
     return [viewScreenShot autorelease];
 }
 
-// Retrieves the XAML FrameworkElement backing this UIView.
+/**
+ Microsoft Extension
+ Retrieves the XAML FrameworkElement backing this UIView.
+*/
 - (WXFrameworkElement*)xamlElement {
-    // Return the CALayerXaml element *backing* this UIView, as opposed to any Xaml element *contained* within the CALayerXaml.
-    // Note: These will be converged into the same Xaml element in the upcoming compositor refactor.
-    return priv->_xamlInputElement.get();
+    return self.layer._xamlElement;
 }
 
-// Sets the backing CALayer's XAML contentsElement (which is null by default)
-// This results in the XAML FrameworkElement being added as a child of this UIView's root CALayer.
-- (void)setXamlElement:(WXFrameworkElement*)xamlElement {
-    [self layer].contentsElement = xamlElement;
+/**
+ Microsoft Extension
+*/
++ (WXFrameworkElement*)createXamlElement {
+    // Default to CALayer's default backing Xaml element
```

Comment:
nope, but i'll change it to '// Use CALayer's default backing Xaml element' since i was overusing 'default' :)

---
In reply to: [87280021](https://github.com/Microsoft/WinObjC/pull/1318#discussion_r87280021) [](ancestors = 87280021)